packages/block-editor/src/components/block-list/block.js

```diff
@@ -83,6 +88,15 @@ function BlockListBlock( {
 	const { removeBlock } = useDispatch( blockEditorStore );
 	const onRemove = useCallback( () => removeBlock( clientId ), [ clientId ] );
 
+	const removeBlockOutline = useSelect(
```

Comment:
Do you think we should move this to `useBlockProps` instead of here (the leaf place where the classname is applied)

Reply:
Oh good catch, `useBlockClassNames` felt like a better fit, so I moved that in https://github.com/WordPress/gutenberg/pull/32567/commits/71d74885c1f0df7e8993654e34175696efd6c47a

Reply:
🤔 the class could maybe use a better name, but I'd like to decouple it from `is-typing` so we can apply it only the situations we need it for.